src/App.js

```diff
@@ -73,6 +74,73 @@ function App() {
     setSearchText("");
   };
 
+  //filter state
+
+  const [withoutOxygen, setWithoutOxygen] = React.useState(false);
+  const [oxygen, setOxygen] = React.useState(false);
+  const [icu, setIcu] = React.useState(false);
+  const [icuwithventilator, setIcuWithVentilator] = React.useState(false);
```

Comment:
I'd do something like, 
```
const defailtState = {
	oxygen: false,
	withoutOxygen: false,
	icu: false,
	icuWithVentilator: false,
}

and create one state instead of many,
const [filterState, setFilterState] = React.useState({...defaultState});
```